rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -338,22 +339,22 @@ where
         &self,
         get_wasm_metadata_payload: GetWasmMetadataRequestPb,
     ) -> Result<Vec<MetadataSection>, String> {
-        let module_hash = <[u8; 32]>::try_from(get_wasm_metadata_payload)?;
+        let hash = <[u8; 32]>::try_from(get_wasm_metadata_payload)?;
```

Comment:
So the method "get_wasm_metadata" won't use the cached values yet, after this PR is merged?  Should we update that?

Reply:
Right. The method won't yet be enabled in production. I'm planning to do what you suggest at the time when we enable it (after the migration).